istioctl/cmd/testdata/v1alpha3/merged-gateway.yaml

```diff
@@ -12,28 +12,28 @@ spec:
   http:
   - match:
     - uri:
-        prefix: /foo/
+        exact: /foo/bar/
```

Comment:
nit: can you revert this? I don't know how the file changes without tests failing, ive seen this on other PRs as well... but probably want to minimize the change